### PR TITLE
Display maintenance status on the app page and apps list

### PR DESCRIPTION
**Pull request by CPatchane**
screen shot 2018-07-31 at 23 53 07

### CODE REVIEW

**Comment by CPatchane** [comment]
[(1) [STO] Afficher les connecteurs en maintenance](https://trello.com/c/6RkJpLLe/1158-1-sto-afficher-les-connecteurs-en-maintenance)

File: src/ducks/apps/index.js [hunk 1]
```diff
@@ -233,12 +233,29 @@ function _consolidateApps(stateApps, newAppsInfos, lang) {
   const apps = new Map()
   stateApps.forEach(app => apps.set(app.slug, app))
   newAppsInfos.forEach(app => {
-    if (app.locales && app.locales[lang]) {
-      extendI18n({ [app.slug]: app.locales[lang] })
-    }
     const appFromState = apps.get(app.slug)
+    // handle maintenance locales
```
Comment (gregorylegarec) [inline src/ducks/apps/index.js]: This whole part can be removed and use only `extendI18n`.

File: src/ducks/apps/index.js [hunk 2]
```diff
@@ -495,6 +495,21 @@ export async function getFormattedRegistryApp(
   // so we removed it from the app manifest if present
   if (manifest.source) delete manifest.source
 
+  // handle locales with maintenance status and messages
+  let appLocales = manifest.locales
```
Comment (gregorylegarec) [inline src/ducks/apps/index.js]: Could be replaced by a call to `extendI18n` IMHO.

**Comment by CPatchane** [comment]
It's updated. I will full use the extendI18n in a next PR, we need this one for an emergency today.
Thanks @gregorylegarec 👍

**Pull request merged**